rspace/src/main/scala/coop/rchain/rspace/history/HistoryRepository.scala

```diff
@@ -61,13 +71,17 @@ object HistoryRepositoryInstances {
       historyStore     = HistoryStoreInstances.historyStore[F](historyLMDBStore)
       history          = HistoryInstances.merging(currentRoot, historyStore)
       // RSpace importer/exporter / directly operates on Store (lmdb)
-      exporter = RSpaceExporterImpl[F](historyLMDBStore, coldLMDBStore, rootsLMDBStore)
-      importer = RSpaceImporterImpl[F](historyLMDBStore, coldLMDBStore, rootsLMDBStore)
+      exporter         = RSpaceExporterImpl[F](historyLMDBStore, coldLMDBStore, rootsLMDBStore)
+      importer         = RSpaceImporterImpl[F](historyLMDBStore, coldLMDBStore, rootsLMDBStore)
+      channelLMDBStore <- StoreInstances.lmdbStore(config.channelHashStore)
```

Comment:
Is this eventLog channel to real channel map? So you already made it persistent?

Reply:
yeah